### ISSUE TITLE
Template Thumbnail does not load when creating new page

### ISSUE THREAD

**Issue by erickgonzalez** [dotCMS/core#19727]
**Describe the bug**
If the selected template contains a thumbnail it should load when creating a new page.

**To Reproduce**
1. Create a new template and add a thumbnail for it.
2. Create a new page and select the template created above.
3. The thumbnail should load, instead throws this error on the log:
```
0:11:41.686  INFO  log.accessLog - Method execution failed: 
com.dotmarketing.business.DotStateException: identifier is null
        at com.dotmarketing.business.IdentifierFactoryImpl.loadFromDb(IdentifierFactoryImpl.java:200) ~[dotcms_master_a6c6506.jar:?]
        at com.dotmarketing.business.IdentifierFactoryImpl.find(IdentifierFactoryImpl.java:474) ~[dotcms_master_a6c6506.jar:?]
        at com.dotmarketing.business.IdentifierAPIImpl.find_aroundBody4(IdentifierAPIImpl.java:83) ~[dotcms_master_a6c6506.jar:?]
        at com.dotmarketing.business.IdentifierAPIImpl$AjcClosure5.run(IdentifierAPIImpl.java:1) ~[dotcms_master_a6c6506.jar:?]
        at org.aspectj.runtime.reflect.JoinPointImpl.proceed(JoinPointImpl.java:149) ~[aspectjrt-1.8.10.jar:?]
        at com.dotcms.aspects.aspectj.AspectJDelegateMethodInvocation.proceed(AspectJDelegateMethodInvocation.java:42) ~[dotcms_master_a6c6506.jar:?]
        at com.dotcms.aspects.interceptors.CloseDBIfOpenedMethodInterceptor.invoke(CloseDBIfOpenedMethodInterceptor.java:29) ~[dotcms_master_a6c6506.jar:?]
        at com.dotcms.aspects.aspectj.CloseDBIfOpenedAspect.invoke(CloseDBIfOpenedAspect.java:41) ~[dotcms_master_a6c6506.jar:?]
        at com.dotmarketing.business.IdentifierAPIImpl.find(IdentifierAPIImpl.java:82) ~[dotcms_master_a6c6506.jar:?]
        at com.dotmarketing.portlets.templates.ajax.TemplateAjax.fetchTemplateImage(TemplateAjax.java:233) ~[dotcms_master_a6c6506.jar:?]
```

Also, check on the Template Properties that old images and new images are shown (templates in the full starter have images).

**Comment by erickgonzalez** [comment]
This is because we are saving the image in the `selectedimage` field instead of the `image` field. We also should remove the `selectedimage` from the TemplateForm since it does not have any uses around the code neither is saved in the DB to avoid confusion.

**Comment by erickgonzalez** [comment]
PR: https://github.com/dotCMS/core/pull/19795

**Comment by hmoreras** [comment]
Core Web changes: https://github.com/dotCMS/core-web/pull/1546/

**Comment by bryanboza** [comment]
Fixed, tested on release-21.02 // Postgres // FF
